### DISCUSSION TITLE
Using MangioRVC models for TTS

### DISCUSSION THREAD

**Discussion by ObsidianStack** [rsxdalv/TTS-WebUI#279] [Q&A]
Hello.
I found ZERO CLUES in how to use mangio rvc models (pth \ index) in TTS Generation WebUi. 
Can someone please help?

**Comment by rsxdalv** [comment]
Are Mangio's models compatible with the regular RVC?

**Reply by ObsidianStack** [reply]
There is no clue about that, and there is no clue in what files it needs the RVC Beta demo to work.

**Reply by rsxdalv** [reply]
So pth is the "model" and index is index.

**Reply by ObsidianStack** [reply]
Yes, and Bark Voice Clone has pth files too but there is no clue in where to put pth files to make the webui see them and use them

**Reply by rsxdalv** [reply]
Are you on Windows?
In gradio you can click the folder icon:
![image](https://github.com/rsxdalv/tts-generation-webui/assets/6757283/c2c2b42f-be5f-495c-879a-ccd517b1eac0)
In React UI the open button:
![image](https://github.com/rsxdalv/tts-generation-webui/assets/6757283/c3be2bda-80b6-4e97-824c-dce5ffe581b6)

**Reply by ObsidianStack** [reply]
I put both files .pth and .index, but it doesn't see any model even hitting the refresh icon near the folder...

**Reply by rsxdalv** [reply]
Ah I see, it requires a folder, like:
MyMangioRVCModels/model.pth
MyMangioRVCModels/index.index
![image](https://github.com/rsxdalv/tts-generation-webui/assets/6757283/e18a01d1-9033-41ad-b095-415207d86347)

**Reply by ObsidianStack** [reply]
Ohh okay, thank you!
